src/components/settings-menu/index.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+
+import { IconLogOut3 } from '@zero-tech/zui/icons';
+import { Address, Avatar, DropdownMenu } from '@zero-tech/zui/components';
+
+import { bem } from '../../lib/bem';
+
+import './styles.scss';
+
+export interface Properties {
+  userName: string;
+  userHandle: string;
+  userAvatarUrl: string;
+
+  onLogout: () => void;
+}
+
+export class SettingsMenu extends React.Component<Properties> {
+  containsAtSymbol = this.props.userHandle.includes('@');
```

Comment:
we don't need to like declare these (eg. `const containsAtSymbol`? ) 😮

Reply:
I think this is just how class components work. I was surprised to lol

Reply:
This is probably a subtle bug, actually. We should turn this into a function. I believe (haven't tested) that if the properties were to change that we'd now have only the original value upon construction.

Reply:
Yep had a quick look into it and that's correct. When initialising class fields like above, they get values once at the time of class instantiation and won't get updated when props change. 

I'll create a PR to transform containsAtSymbol and userHandle into methods.